src/components/SettingsComponent.js

```diff
@@ -63,7 +70,7 @@ function SettingsComponent({ settingsOptions }) {
 								<View
 									style={{
 										height: 1,
-										backgroundColor: colors.grey
+										backgroundColor: colors.card
```

Comment:
This background color is used for the horizontal line.
Making it the same color as the ScrollView would make it invisible.